ice/cli/websocket/server.zep

```diff
@@ -0,0 +1,328 @@
+
+namespace Ice\Cli\Websocket;
+
+use Ice\Exception;
+
+/**
+ * Websocket server.
+ *
+ * @package     Ice/Cli
+ * @category    Component
+ * @author      Ice Team
+ * @copyright   (c) 2014-2018 Ice Team
+ * @license     http://iceframework.org/license
+ */
+class Server extends Websocket
+{
+
+    protected verbose = false { set };
+    protected address;
+    protected server { get };
+    protected sockets = [];
+    protected clients = [] { get };
+    protected callbacks = [];
+
+    /**
+     * Create an instance.
+     *
+     * @param string address Where to create the server, defaults to "ws://127.0.0.1:8080"
+     * @param array options Stream context options
+     */
+    public function __construct(string address = "ws://127.0.0.1:8080", array options = [])
+    {
+        var addr, context, key, value;
+
+        let addr = parse_url(address);
+
+        if addr === false || !isset addr["scheme"] || !isset addr["host"] || !isset addr["port"] {
+            throw new Exception("Invalid address");
+        }
+
+        let context = stream_context_create();
+
+        if count(options) {
```

Comment:
if options {

Reply:
I can't do that in zephir _Variable can't be evaluated array_.